src/ai/backend/kernel/intrinsic.py

```diff
@@ -148,9 +148,16 @@ async def prepare_sshd_service(service_info):
 
 async def prepare_ttyd_service(service_info):
     shell = "sh"
-    if Path("/bin/bash").exists():
+    if Path("/bin/zsh"):
+        shell = "zsh"
+        zshrc_path = Path("~/.zshrc").expanduser()
+        with zshrc_path.open("w") as f:
+            f.write("export PATH=$PATH:/opt/backend.ai/bin\n")
+            f.write("export TERM=xterm-256color\n")
+            f.write("autoload -Uz compinit && compinit\n")
```

Comment:
I think this should be done automatically. Could you explain why this is required for zsh?

Reply:
I think this command is not always enabled by default in Zsh, that's why I added this line